ComposeAdvanced/gradle/libs.versions.toml

```diff
@@ -1,11 +1,12 @@
 [versions]
 android-gradle-plugin = "7.1.3"
-androidx-activity = "1.4.0"
-androidx-lifecycle = "2.4.1"
+androidx-activity = "1.5.0-beta01"
```

Comment:
I figured this should be safe since they are beta, should be final before wear-compose?